### PR TITLE
Do not show WebXR icon in the content feed

### PR DESCRIPTION
**Pull request by MortimerGoro**

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java [hunk 1]
```diff
@@ -285,6 +289,16 @@ public void onChanged(Spannable aUrl) {
         }
     };
 
+    private Observer<Spannable> mIsContentFeedObserver = new Observer<Spannable>() {
+        @Override
+        public void onChanged(Spannable o) {
+            String current = url.getValue().toString();
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java]: Where does `url` come from?

Comment (bluemarvin) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java]: Okay, I think I see. The variable naming in this file is all over the place. We have a member variable name `url` but also local variables called `url`. This file really needs to be cleaned up.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java [hunk 2]
```diff
@@ -285,6 +289,16 @@ public void onChanged(Spannable aUrl) {
...
+            String feed = getApplication().getString(R.string.homepage_url);
```
Comment (keianhzo) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java]: Can we move this to UrlUtils? We already have some helper methods there for identifying specific URLs.

Comment (MortimerGoro) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/viewmodel/WindowViewModel.java]: Done

**Pull request merged**